src/App.jsx

```diff
@@ -16,6 +17,10 @@ function App() {
 					path="/"
 					element={<Home />}
 				/>
+				<Route
+					path="/articles/:article_id"
+					element={<Article />}
+				/>
 			</Routes>
```

Comment:
it's fine having the route for articles/:article_id but one massive issue is that i have no way of getting to the article via the articles list which is the most logical way for me to do it